scripts/mafiachecker.js

```diff
@@ -351,11 +363,17 @@ function mafiaChecker() {
                             if (commandList.indexOf("inspect") !== -1)
                                 commonOptional = commonOptional.concat(["Sight", "inspectMsg"]);
                             if (commandList.indexOf("distract") !== -1)
-                                commonOptional = commonOptional.concat(["distractmsg", "teammsg", "onlyUser"]);
+                                commonOptional = commonOptional.concat(["distractmsg", "teammsg", "onlyUser", "distractActions"]);
                             if (commandList.indexOf("protect") !== -1)
                                 commonOptional = commonOptional.concat(["protectmsg"]);
+                            if (commandList.indexOf("bomb") !== -1)
+                                commonOptional = commonOptional.concat(["bombMsg"]);
                             if (commandList.indexOf("dayprotect") !== -1)
                                 commonOptional = commonOptional.concat(["dayProtectMsg", "dayProtectType"]);
+                            if (commandList.indexOf("daydistract") !== -1)
+                                commonOptional = commonOptional.concat(["dayDistractMsg", "dayDistractType"]);
+                            if (commandList.indexOf("voteblock") !== -1)
+                                commonOptional = commonOptional.concat(["voteBlockMsg", "voteBlockDuration"]);
```

Comment:
Ignore these but yu have a lot of other error. 

What kind of text editor do you use? Notepad++ has a checker you can add very easily